cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PushFilter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2016 IBM Corp. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ *  except in compliance with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the
+ *   License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  either express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+package com.cloudant.sync.replication;
+
+import com.cloudant.sync.datastore.DocumentRevision;
+
+/**
+ * Created by Rhys Short on 31/03/2016.
```

Comment:
Fixed in 8e2cd25. Also added a description of the interface and an example usage.